### PR TITLE
[Learn]: Added Pagination for chapters

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**
This PR adds pagination for chapters under learn-ng.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Note: Few unrelated files are due to lint fixes while committing, please ignore them.

**Comment by l5io** [comment]
🚀 Preview for commit 74e27d359c6ea4b45e2d0109576e4f6ca93ce15b at: https://612b79ec241ae311738f6b71--layer5ng.netlify.app

File: src/components/Learn-Components/TOC-Chapters/index.js [hunk 1]
```diff
@@ -49,7 +40,7 @@ const TOC = ({ TOCData,courseData, chapterData, location }) => {
           {availableChapters.map((item) => (
             <li key={item} className={item === path ? "active-link" : ""}>
               <p className="toc-item">
-                <a href={`/learn-ng/${chapterData.fields.learnpath}/${chapterData.fields.course}/${getActiveServiceMesh()}/${item}/`}>
+                <a href={`/learn-ng/${chapterData.fields.learnpath}/${chapterData.fields.course}/${getActiveServiceMesh(chapterData)}/${item}/`}>
```
Comment (Tanuj22) [inline src/components/Learn-Components/TOC-Chapters/index.js:43]: Can't this be a `Link` ?

File: src/components/Learn-Components/getActiveServiceMesh.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const getActiveServiceMesh = (chapterData) => chapterData.fields.slug.split("/")[3];
```
Comment (Tanuj22) [inline src/components/Learn-Components/getActiveServiceMesh.js:1]: Better to keep in `utils`?

File: src/components/Learn-Components/getCurrentPage.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const getCurrentPage = (location) => {
```
Comment (Tanuj22) [inline src/components/Learn-Components/getCurrentPage.js:1]: Same as above

File: src/sections/Learn-Layer5/Chapters/index.js [hunk 4]
```diff
@@ -51,20 +52,21 @@ const Chapters = ({chapterData, courseData, location, serviceMeshesList, TOCData
...
+  const capitalize = (s) => s.charAt(0).toUpperCase() + s.slice(1);
 
   const ServiceMeshesAvailable = ({serviceMeshes}) => serviceMeshes.map((sm, index) => {
 
     return(  
       <>
-        <a  href={`/${sm.slug}`} data-for="mesh-name" data-tip={sm.section} className="course" key={sm+index}>
-          <div className={`service-mesh-image ${isMeshActive(sm.section) ? "service-mesh-image-active" : ""}`}>
+        <div className={`service-mesh-image ${isMeshActive(sm.section) ? "service-mesh-image-active" : ""}`}>
+          <a  href={`/${sm.slug}`} data-for="mesh-name" data-tip={capitalize(sm.section)} className="course" key={sm+index}>
```
Comment (Tanuj22) [inline src/sections/Learn-Layer5/Chapters/index.js:62]: `Link`?

**Comment by Nikhil-Ladha** [comment]
Made the changes.

**Review by Tanuj22 — APPROVED** [review]

**Pull request merged**